### PR TITLE
Update README to be more welcoming 

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
This patch:
- Updates the readme making it more welcoming and providing quick start instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,50 @@
...
 
-The Node Feature Discovery operator manages detection of hardware features and configuration in a kubernetes cluster.
-The operator orchestrates all resources needed to run the [Node-Feature-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) DaemonSet
+The Node Feature Discovery operator is a tool for Kubernetes administrators that makes it easy to detect and understand the hardware features and configurations of a cluster's nodes. With this operator, administrators can easily gather information about their nodes that can be used for scheduling, resource management, and more by controlling the life cycle of the [Node-Feature-Discovery](https://github.com/kubernetes-sigs/node-feature-discovery) DaemonSet.
 
-Welcome to Node Feature Discovery Operator!
+## How it Works
 
-### See our [Documentation][documentation] for detailed instructions and reference
+The operator works by orchestrating all resources needed to run the Node-Feature-Discovery DaemonSet. This DaemonSet runs on each node in the cluster and detects the features and configurations of the node's hardware. The operator then collects and organizes this information, making it easily accessible.
```
Comment (marquiz) [inline README.md:7]: I'd suggest to drop the term daemonset here as it's nfd-worker that is run as a daemonset and nfd-master that is a deployment

```suggestion
The operator works by orchestrating all resources needed to run the Node-Feature-Discovery (NFD). NFD runs on each node in the cluster and detects the features and configurations of the node's hardware ...
```

```plaintext
The operator then collects and organizes this information, making it easily accessible.
```

What information? Node features(?)  This is a bit confusing. It's NFD which makes them accessible, not the operator.

**nit:** also, I'd suggest to wrap the lines to e.g. 80 characters for easier readability/editability in text editors.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,50 @@
...
 
-## Community, discussion, contribution, and support
+## Quick start
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+Get the source code
```
Comment (marquiz) [inline README.md:11]: Do we want to advertise this as the preferred deployment method? How about just

```
kubectl create -f https://operatorhub.io/install/nfd-operator.yaml
```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,50 @@
...
+
+For more detailed information on how to use the Node Feature Discovery operator, please check out our [documentation](https://github.com/kubernetes-sigs/node-feature-discovery-operator/blob/master/docs/index.md)
+
+## Contributing
+
+The Node Feature Discovery operator welcomes contributions, and interested parties are encouraged to take a look at the [contributing guidelines](CONTRIBUTING.md) and [open issues](https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues). We're excited to have you join our community of contributors.
+
+## Support
+
+If there are any issues or questions about the Node Feature Discovery operator, they can be addressed by opening an issue on the [GitHub repository](https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues) or reaching out on the [Slack channel](https://kubernetes.slack.com/messages/node-feature-discovery-operator).
```
Comment (marquiz) [inline README.md:50]: I don't thing we have a dedicated slack channnel for the operator only

File: SECURITY_CONTACTS [hunk 4]
```diff
@@ -10,5 +10,6 @@
 # DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
 # INSTRUCTIONS AT https://kubernetes.io/security/
 
+ArangoGutierrez
```
Comment (marquiz) [inline SECURITY_CONTACTS:13]: I suggest to have this as a separate PR

File: README.md (hunk 3 above)
Comment (marquiz) [inline README.md:50]: You could point to https://github.com/kubernetes-sigs/node-feature-discovery-operator/issues/new/choose for new issues

File: OWNERS [hunk 5]
```diff
@@ -3,4 +3,5 @@
 approvers:
   - ArangoGutierrez
   - marquiz
+reviewers:
```
Comment (marquiz) [inline OWNERS:6]: I'd suggest to handle zvonkok's demotion in a separate PR

**Review by marquiz — COMMENTED** [review]
Thanks @ArangoGutierrez for the improvement. I had a few comments/suggestions.

One format-related wish (I think I mentioned that in some comment) is that please wrap the lines to some meaningful lenght (e.g. the standard 79/80 chars). Would make editing and also commenting a lot easier.

File: README.md (hunk 2 above)
Comment (ArangoGutierrez) [reply README.md:11]: this is the same `Quick start` from the documentation pages, that way you point at requires an extra step to enable the operatorhub on the user cluster.

I am basically bringing the quick start section from the documentation to the readme for a "quick start"

Comment (ArangoGutierrez) [reply README.md:20]: that's where I am trying to move, I want to have HELM charts for the operator, been busy with other stuff, but I am planning to add HELM soon

**Comment by ArangoGutierrez** [comment]
@marquiz PTAL :)

**Review by marquiz — APPROVED** [review]
Thanks @ArangoGutierrez, let's merge this and improve deployment options later 😊
/lgtm

**Pull request merged**